core/src/main/java/io/github/mabartos/engine/DefaultRiskEngine.java

```diff
@@ -102,6 +103,11 @@ public void evaluateRisk(RiskEvaluator.EvaluationPhase evaluationPhase) {
 
     @Override
     public void evaluateRisk(RiskEvaluator.EvaluationPhase phase, RealmModel realm, UserModel knownUser) {
+        if (!isRiskBasedAuthnEnabled()) {
+            logger.debug("Risk-based authentication is disabled. Skipping risk evaluation.");
```

Comment:
```suggestion
            logger.warn("Risk-based authentication is disabled. Skipping risk evaluation.");
```

We can consider the Risk engine evaluation is happening directly in the authentication flow, so it means the risk engine is already present in the used flow. 

Warning seems more reasonable to me.